src/routes/about/officer-profile.svelte

```diff
@@ -10,40 +17,150 @@
   export let placeholderPicture = 'placeholder.webp';
   export let dev = false;
 
+  // earliestTerm returns the earliest term of the current officer. Blame Ethan
+  // for this whole idea of having an officer ID. OK, maybe some of it was mine.
+  function earliestTerm(): string {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    for (const [_, key] of Object.entries(Term)) {
+      if (info.positions[key]) {
+        return key;
+      }
+    }
+    return 'null';
+  }
+
+  const officerID = info.fullName.toLowerCase().replace(/[^a-z0-9]/g, '-') + '-' + earliestTerm();
   const officerName = info.fullName ?? '';
   const officerPicture = info.picture ?? placeholderPicture;
+  const officerSocials = info.socials ?? {};
+  $: officerPosition = getPositionByTermIndex(info, $termIndex)?.title || '';
+
+  /** @see <https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText> */
+  function copyDiscord() {
+    toastDo(() => navigator.clipboard.writeText(officerSocials.discord || ''), {
+      success: () => `Copied ${officerName}'s Discord tag to clipboard!`,
+      failure: () => 'Error occured while copy Discord tag to clipboard.',
+    });
+  }
+
+  const positionClasses = Object.entries({
```

Comment:
Nit: Provide a comment to clarify how this variable is used. 

Note: The value of positionClasses should be derived from a future common module containing predictable information about each team (related: #538).